### DISCUSSION TITLE
Not all items visible to members of organization in shared collection.

### DISCUSSION THREAD

**Discussion by parski** [dani-garcia/vaultwarden#3741] [Q&A]
### Deployment environment

* vaultwarden version:
1.29.0

* Install method: 
Docker compose file.
* Clients used: 
iOS
* Reverse proxy and version: 
image: caddy:2
* MySQL/MariaDB or PostgreSQL version: 
SQLite
* Other relevant details:
Ubuntu
### Steps to reproduce

I start vaultwarden with `docker compose up -d`.

1) Create organization, invite people (IS, KF, LS, LI, MS, PA), make sure their accounts are verified and they are apart of the organization.
2) Set all members to Admin. They are all "Can Edit" in the shared collection.
3) I, PA, add items to the collection. Everyone sees them. This was May 5 2020.
4) Years later on Dec 24 2022, KS adds items to the collection. Only they, me and IS see the items. I find this out today, Jul 26 2023.
### Expected behaviour

I except everyone in the organization to see the items shared by KS and anyone else in the organization.
### Actual behaviour

Only me, KS and IS see the items added by KS. Everyone else has been syncing their vaults, updating their client and using Bitwarden daily since KS added the items and long before that. They have just never been aware that KS added the items and I've assumed they've all had access all this time.
### Troubleshooting data
<img width="575" alt="Screenshot 2023-07-26 at 21 50 18" src="https://github.com/dani-garcia/vaultwarden/assets/2445818/112db650-81f1-4e66-b1a6-6a9c52d46ea9">

<img width="1069" alt="Screenshot 2023-07-26 at 22 02 35" src="https://github.com/dani-garcia/vaultwarden/assets/2445818/b9f6f1da-f775-4b8f-8dd6-8432b2125b78">

**Comment by BlackDex** [comment]
Did they refreshed there vault? Logout/login to be sure?
If all members are admin, and all are part of the same collection they should see the same ciphers.

Edit:
The reason is, the ciphers are not tight to users, but collections. And if they are able to see other ciphers of that collection, they should see the new one also.

Also, be sure to check it via the Organization interface, that should also sync the org ciphers again.

**Comment by BlackDex** [comment]
@parski any update?

**Comment by parski** [comment]
It seems to have synced now after logging out and in manually. What I don't get is how none of their vaults synced in more than half a year but that makes it a client issue, not a Vaultwarden issue. 

You can close this. Thanks for checking in.

**Comment by BlackDex** [comment]
That is strange indeed, since the clients normally sync each x amount of time, or at least check if that is needed and if so, execute that sync.

It could be that they had an old client, cached in the browser, and then a server update somehow broke that sync and then also somehow not logged those users out, which it should have done.

Maybe a chain of several items. But glad it is synced now at least :).
